singer/src/main/java/com/pinterest/singer/writer/KafkaWriter.java

```diff
@@ -221,6 +233,11 @@ public HeadersInjector getHeadersInjector() {
     return headersInjector;
   }
 
+  @VisibleForTesting
+  public ExecutorService getClusterThreadPool() {
+    return clusterThreadPool;
+  }
+
```

Comment:
Could we remove the method `includeAuditMessageInBatch` as it is no longer used.

Reply:
removed in the update of the PR.